cookbook/frontend/assetic_php.rst

```diff
@@ -0,0 +1,152 @@
+.. index::
+    single: Front-end; Assetic, Bootstrap
+
+Combining, Compiling and Minimizing Web Assets with PHP Libraries
+=================================================================
+
+The official Symfony Best Practices recommend to use Assetic to
+:doc:`manage web assets </best_practices/web-assets>`, unless you are
+comfortable with JavaScript-based frontend tools.
+
+Even if those JavaScript-based solutions are the most suitable ones from a
+technical point of view, using pure PHP alternative libraries can be useful in
+some scenarios:
+
+* If you can't install or use ``npm`` and the other JavaScript solutions;
+* If you prefer to limit the amount of different technologies used in your
+  applications;
+* If you want to simplify application deployment.
+
+In this article you'll learn how to combine and minimize CSS and JavaScript files
```

Comment:
It's 3:

> 
Use a comma to set off introductory elements, as in "Running toward third base, he suddenly realized how stupid he looked."

> It is permissible to omit the comma after a brief introductory element if the omission does not result in confusion or hesitancy in reading. If there is ever any doubt, use the comma, as it is always correct. If you would like some additional guidelines on using a comma after introductory elements, click HERE.

So, yes. You are correct that the comma can be omitted, but it can introduce confusion. As the docs are read by many non-natives, it's good to avoid anything that could introduce confusion.